src/api/component.js

```diff
@@ -154,7 +154,7 @@ export default class extends HTMLElement {
     // static render()
     // Note that renderCallback is an optional method!
     if (!this.renderCallback && constructor.render) {
-      deprecated(this, 'static render', 'renderCallback');
+      DEBUG && deprecated(this, 'static render', 'renderCallback');
```

Comment:
`DEBUG` is much easier to type and we need to have it at the call site of the thing we want to do so that it gets removed during dead code elimination.